packages/core/src/config/load.ts

```diff
@@ -102,3 +108,20 @@ export async function getConfig(configPath: string | undefined = findConfig()):
     throw new Error(`Error parsing config file at '${configPath}': ${e.message}`);
   }
 }
+
+interface CreateConfigOptions {
```

Comment:
We usually use `type` notation as more general.

```suggestion
type CreateConfigOptions = {
```